src/main/java/chess/controller/Controller.java

```diff
@@ -1,48 +1,76 @@
 package chess.controller;
 
+import static spark.Spark.get;
+import static spark.Spark.post;
+import static spark.Spark.staticFiles;
+
+import chess.dao.BoardDao;
 import chess.domain.boardstrategy.InitBoardStrategy;
 import chess.domain.game.ChessGame;
-import chess.domain.game.state.Init;
-import chess.domain.game.state.State;
-import chess.domain.game.state.attribute.StateType;
-import chess.dto.BoardDto;
+import chess.dto.ChessGameDto;
 import chess.dto.CommandDto;
-import chess.view.InputView;
-import chess.view.OutputView;
+import java.util.HashMap;
+import java.util.Map;
 
 public class Controller {
+    private static final BoardDao boardDao = new BoardDao();
+    private static final RenderService service = new RenderService();
```

Comment:
변수 네이밍은 명확한게 좋을 것 같습니다.
```suggestion
    private static final RenderService renderService = new RenderService();
```